testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2269,6 +2271,7 @@ private List<SetColumnTypeSetup> setColumnTypeSetupData()
                 .add(new SetColumnTypeSetup("char(100)", "'shorten-char'", "char(50)", "cast('shorten-char' as char(50))"))
                 .add(new SetColumnTypeSetup("char(100)", "'char-to-varchar'", "varchar", "'char-to-varchar'"))
                 .add(new SetColumnTypeSetup("varchar", "'varchar-to-char'", "char(100)", "cast('varchar-to-char' as char(100))"))
+                .add(new SetColumnTypeSetup("row(x int)", "row(1)", "row(y int)", "cast(row(1) as row(y int))"))
```

Comment:
I am not sure this should be the expected behavior.
I would rather map row fields by name: the `x` field is gone, the `y` field is new, it should have a null value.

Reply:
Changed to `row(x int) -> row(x bigint)` instead.